keyboards/synthlabs/solo/solo.c

```diff
@@ -0,0 +1 @@
+#include "solo.h"
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

Reply:
Added as of 48aa8c696efe739a80c490b2883d5c324945690e.

Reply:
Whoops, sorry. Missed that one - thanks for the catch. Added header as of ac7f06b761afa713891120800f66418198d1b24d.